code/src/index.css

```diff
@@ -11,3 +14,24 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.container {
+  max-width: 1400px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  align-content: center;
+  margin: auto;
+}
+
+/* Media queries  */
+
+@media only screen and (max-width: 420px) {
+  .container {
+    max-width: 300px;
+  }
+}
+
+/* Media queries  */
```

Comment:
So nice with these different css files, makes it so much easier to understand what the code does!